File: AspNetCore_gdr2333_Front+Back/Back/Types/DataBase/TreehollowComment.fs
namespace Back.Types.DataBase

open System
open System.ComponentModel.DataAnnotations

type TreehollowComment(treehollowId: int64, senderId: int64, content: string) =
    class
        [<Key>]
        member val CommentId = 0L with get

        member val TreehollowId = treehollowId with get
        member val SenderId = senderId with get
        member val Content = content with get
        member val SendTime = DateTime.Now with get
        member val Checked = false with get, set
        member val CheckSucess = false with get, set
    end

File: AspNetCore_gdr2333_Front+Back/Back/Types/DataBase/TreehollowComment.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Back.Types.DataBase;

public class TreehollowComment
{
    [Key]
    public long CommentId { get; set; }

    public long TreehollowId { get; set; }

    public long SenderId { get; set; }

    public string Content { get; set; }

    public DateTime SendTime { get; set; }

    public bool Checked { get; set; }

    public bool CheckSuccess { get; set; }

    public bool Deleted { get; set; }

    public TreehollowComment(long treehollowId, long senderId, string content)
    {
        TreehollowId = treehollowId;
        SenderId = senderId;
        Content = content;
        SendTime = DateTime.Now;
        Checked = false;
        CheckSuccess = false;
        Deleted = false;
    }

    public TreehollowComment()
    {
    }
}
